mos_tests/rabbitmq_oslo/conftest.py

```diff
@@ -38,3 +44,57 @@ def fixt_kill_rpc_server_client(env):
         with node.ssh() as remote:
             cmd = 'pkill -f oslo_msg_check_'
             remote.execute(cmd)
+
+
+@pytest.yield_fixture
+def fixt_iptables_drop_reject(env, request):
+    """Choose one controller and apply IPTables rules"""
+    i_am_chosen_mark = '/tmp/i_am_chosen_mark.txt'
+
+    if request.param == 'drop':
+        table_modif = (
+            'iptables -I INPUT -p tcp -m tcp --dport 4369  -j DROP ; '
```

Comment:
Let's make 2 command templates like `iptables -{action} INPUT -p tcp -m tcp --dport {port}  -j DROP`
And do smth like `remote.execute(cmd_template.format(action='I', port=4369))` (perhaps in loop over ports)